### ISSUE TITLE
[Issue] With Terrablender installed, JAMD overworld mining dim does not spawn ores

### ISSUE THREAD

**Issue by muggleorwizard** [nanite/JAMD#33]
### Minecraft Version

1.21

### Modloader

neoforge-21.0.167

### Mod version

neo-5.0.3+mc1.21

### Latest version?

- [X] Yes

### Repeatable?

- [X] Yes

### Link to log files

N/A

### Steps to reproduce the issue

Install a neoforge instance with Terrablender 1.21-4.0.0.1 and JAMD neo-5.0.3+mc1.21. Enter the overworld mining dimension, dig down.

### Description of the issue you are having

When entering the overworld mining dimension of JAMD with Terrablender installed, there will be no ores generated in the mining dimension. JAMD is functional when Terrablender is not installed. Issue also created on the Terrablender GitHub: Glitchfiend/TerraBlender#174

### Information

- [X] I have provided as much information as possible

**Comment by UnRealDinnerbone** [comment]
Can you upload the config/jamd/mining.json So I can see what it looks like
What JAMD does it scan for all ore features on the first load them adds them to that json. And that JSON is what is worldgen is for JAMD

**Comment by muggleorwizard** [comment]
[mining.json](https://github.com/user-attachments/files/16606625/mining.json)

**Comment by UnRealDinnerbone** [comment]
The ore are in the json to spawn, guessing the tag minecraft:stone_ore_replaceables is getting changed

**Comment by muggleorwizard** [comment]
yeah, that's why I think Terrablender may be the issue. I looked at past Terrablender issues and there was a similar example of this problem on 1.18 I believe: https://github.com/Glitchfiend/TerraBlender/issues/94

**Comment by MokahTGS** [comment]
Um is there a fix for this as this is an issue in 1.20.1 as well.

**Comment by muggleorwizard** [comment]
this is still an issue

**Comment by Fr3shlama** [comment]
any updates?
